src/ripple/app/main/Application.cpp

```diff
@@ -1105,7 +1105,12 @@ bool ApplicationImp::setup()
     m_overlay->setupValidatorKeyManifests (*config_, getWalletDB ());
 
     {
-        auto setup = setup_ServerHandler(*config_, std::cerr);
+        std::stringstream parseErr;
+        auto setup = setup_ServerHandler(*config_, parseErr);
+        if(parseErr.rdbuf()->in_avail())
+        {
+            JLOG(m_journal.error()) << parseErr.str();
```

Comment:
@mellery451 I think it would be better if `m_journal.error()` was convertible to `std::ostream&`, or if `class Stream` had a member which returned a suitable `std::ostream`. Forcing callers to declare `std::stringstream` is clumsy.